src/main.cpp

```diff
@@ -93,7 +95,9 @@ int main(int argc, const char *argv[])
 
     global_video[0] = std::make_shared<video_stream>(0, &cfg->stream0, "stream0");
     global_video[1] = std::make_shared<video_stream>(1, &cfg->stream1, "stream1");
-    global_jpeg = std::make_shared<jpeg_stream>(jpeg_stream{2, &cfg->stream2});
+    global_jpeg[0] = std::make_shared<jpeg_stream>(2, &cfg->stream2);
+    //global_jpeg[1] = std::make_shared<jpeg_stream>(jpeg_stream{3, &cfg->stream3});
```

Comment:
Is this line necessary?

Reply:
in preparation to have more than 1 JPEG channel in future this placeholder was added and should not harm.
And no it's not necessary.